### ISSUE TITLE
Problematic edge case with QuasiQuotes

### ISSUE THREAD

**Issue by torgeirsh** [ndmitchell/record-dot-preprocessor#46]
Using Text.URI.QQ from the modern-uri package, [uri|test.com|] works as expected, but [uri|test.com/|] fails to compile:
```
  |
1 | (Z.getField @"com" test)/
  |            ^
unexpected space
expecting '#', '%', '/', ':', '?', '@', end of input, sub-delimiter, or unreserved character

    • In the quasi-quotation: [uri|(Z.getField @"com" test)/|]
```
It seems like the trailing slash prevents the preprocessor from detecting the quasi-quote.

**Comment by ndmitchell** [comment]
Thanks for the report. I've pushed a fix - can you confirm that does it for you? I had to do this one sight unseen, because my compiler is currently screwed. If it appears to work for you I'll make a release.

**Comment by torgeirsh** [comment]
I've done a quick test, and it does indeed seem to work. Thank you!

**Comment by ndmitchell** [comment]
Thanks for testing - released as 0.2.13
